terraform/deployments/ga4-analytics/project_iam_binding.tf

```
@@ -0,0 +1,75 @@
+resource "google_project_iam_binding" "project-owners" {
+  project = google_project.project.project_id
+  role    = "roles/owner"
+
+  members = [
+    "group:gcp-ga4-analytics-352613-owners@digital.cabinet-office.gov.uk",
+    "serviceAccount:terraform-cloud-production@govuk-production.iam.gserviceaccount.com"
+  ]
+}
+
+resource "google_project_iam_binding" "project-editors" {
+  project = google_project.project.project_id
+  role    = "roles/editor"
+  members = [
+    "group:gcp-ga4-analytics-352613-editors@digital.cabinet-office.gov.uk",
+  ]
+}
+
+resource "google_project_iam_binding" "project-viewers" {
+  project = google_project.project.project_id
+  role    = "roles/viewer"
+  members = [
+    # purposfully empty
+  ]
+}
+
+resource "google_project_iam_binding" "project-gds_bigquery_editor" {
+  project = google_project.project.project_id
+  role    = "projects/ga4-analytics-352613/roles/GDS_BQ_editor"
+  members = [
+    "serviceAccount:firebase-measurement@system.gserviceaccount.com",
+    "serviceAccount:search-console-data-export@system.gserviceaccount.com",
+    "serviceAccount:service-177535650450@gcp-sa-dataform.iam.gserviceaccount.com",
+    "serviceAccount:ga4-user-admin@ga4-analytics-352613.iam.gserviceaccount.com",
+    "serviceAccount:ga-database@ga4-analytics-352613.iam.gserviceaccount.com",
+    "serviceAccount:service-659461823838@gcp-sa-dataform.iam.gserviceaccount.com",
+    "serviceAccount:177535650450-compute@developer.gserviceaccount.com",
+  ]
+  depends_on = [
+    google_service_account.sa--ga-database,
+    google_service_account.sa--ga4-user-admin,
+  ]
+}
+
+resource "google_project_iam_binding" "project-gds_bigquery_user" {
+  project = google_project.project.project_id
+  role    = "projects/ga4-analytics-352613/roles/gds.bigquery.user"
+  members = [
+    "domain:digital.cabinet-office.gov.uk",
+    "user:arran.gosal@merkle.com",
+  ]
+}
+
+resource "google_project_iam_binding" "project-GDS_BQ_read_access" {
```

```suggestion
resource "google_project_iam_binding" "project-gds-bq-read-access" {
```

Can we at least normalise the names for the terraform resources?

---

yes, much cleaner and easier to read